db/schema.rb

```diff
@@ -1216,9 +1216,9 @@
     t.datetime "updated_at", null: false
```

Comment:
this file should not be part of this PR, please remove it.

Reply:
Removed it in [this](https://github.com/thepracticaldev/dev.to/pull/7103/commits/bea5bdd2efec42579c18e07618c604d33b7511a7#diff-1acd2e7e27a227829d5d14a91c863bb6L1216) commit.

Reply:
I can still see some updates in that `schema.rb` file. I think you may need to just drop this file from your PR.

Reply:
@ludwiczakpawel  reverted the changes done in the schema file. Please verify now.